commands.js

```diff
@@ -64,6 +64,28 @@ class Commands {
     selfDestructMessage(() => message.reply('added time to user.'))
   }
 
+  async bitrate (message) {
+    let args = message.split(' ').splice(1).filter(str => str !== '')
+    let mem = message.member
+    let channel = message.member.voiceChannel
+    if (channel == null) {
+      throw new Error(`<@${message.author.id}>! You aren't in a voice channel`)
+    }
+    if (args.length === 0) {
+      selfDestructMessage(() => message.reply(`<#${channel.id}> bitrate: ` + channel.bitrate))
+    } else {
+      if (isNaN(args[0])) {
+        throw new Error(`<@${message.author.id}>! Input is not a valid bitrate(i.e. 64000)`)
+      }
+      if (channel.locked_by === mem) {
```

Comment:
I'm not sure I understand this. If the channel is locked by the calling user, we tell them it's not a valid voice channel? (Also, `locked_by` contains the id of the member, not the member object itself. Consider comparing against `message.author.id` instead - then you don't even need to keep `mem` around.)